include/beast/http/impl/write.ipp

```diff
@@ -527,11 +538,16 @@ write(SyncWriteStream& stream,
     wp.sb.consume(wp.sb.size());
     if(! result)
     {
+        detail::writef_lambda<SyncWriteStream> wf{
```

Comment:
Looking at the tip of develop, I still see the `#if 0` block. https://github.com/vinniefalco/Beast/blob/develop/include/beast/http/impl/write.ipp#L545-L550